CVLByExample/Curve/README.md

```diff
@@ -0,0 +1,35 @@
+
+#Curve 
+
+This directory contains the `Curve` contract which has the read only reentrancy weakness. 
+
+##Contracts
+The original contract is `Curve`` whose weakness is found by the builtin rule viewReentrancy. In addition we have the
+contract MungedCurve that allows finding the weakness also with a regular rule.
+
+##Specs
+`certora/specs`` contains two spec files for exposing this weakness.
+
+1. `BuiltinViewReentrancy.spec` uses the builtin rule `viewReentrancy` that checks that for every solidity function and every
```

Comment:
Again, it's not finding it on Curve, but on our Curve Mock. It's important to distinguish it since we don't want to imply Curve are misbehaving, they already mitigated this somewhat on their new version. We're trying to say that we can find it on similar cases (or even a similar weakness on a not so similar codebase.